pkgs/games/jazz2/default.nix

```diff
@@ -0,0 +1,46 @@
+{ cmake
+, fetchFromGitHub
+, glew
+, glfw
+, lib
+, libGL
+, libopenmpt
+, libvorbis
+, openal
+, SDL2
+, stdenv
+, xorg
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jazz2";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "deathkiller";
+    repo = "jazz2-native";
+    rev = version;
+    sha256 = "sha256-VJqF6fvg3kYVl4zWuo4z4QuDFRGWq5B4N0u/cI7jaKo=";
+  };
+
+  buildInputs = [ libGL SDL2 zlib glew glfw openal libvorbis libopenmpt ];
+  nativeBuildInputs = [ cmake ];
+  propagatedBuildInputs = [ xorg.libSM xorg.libICE xorg.libXext ];
```

Comment:
Why do we need to propagate those?

Reply:
It crashed without xorg.libSM so before adding only this one and rebuilding - I checked executable with ldd and it was showing that these 3 were missing